larva/src/main/java/org/frankframework/larva/queues/QueueUtils.java

```diff
@@ -39,30 +39,29 @@
 public class QueueUtils {
 	private static final Logger LOG = LogUtil.getLogger(QueueUtils.class);
 
-	public static IConfigurable createInstance(ClassLoader classLoader, String className) {
+	public static IConfigurable createInstance(IbisContext ibisContext, ClassLoader classLoader, String className) {
 		ClassLoader originalClassLoader = Thread.currentThread().getContextClassLoader();
 		Thread.currentThread().setContextClassLoader(classLoader);
 		try {
-			return createInstance(className);
+			return createInstance(ibisContext, className);
 		} finally {
 			if (originalClassLoader != null) {
 				Thread.currentThread().setContextClassLoader(originalClassLoader);
 			}
 		}
 	}
 
-	private static IConfigurable createInstance(String className) {
+	private static IConfigurable createInstance(IbisContext ibisContext, String className) {
 		LOG.debug("instantiating queue [{}]", className);
 		try {
 			Class<?> clazz = ClassUtils.loadClass(className);
-			Constructor<?> con = ClassUtils.getConstructorOnType(clazz, new Class[] {});
-			Object obj = con.newInstance();
+			Object obj = ibisContext.createBeanAutowireByName(clazz);
 
-			if(obj instanceof NameAware object) { //Set the name
+			if (obj instanceof NameAware object) { // Set the name
 				object.setName("Test Tool "+clazz.getSimpleName());
 			}
 
-			if(obj instanceof AbstractHttpSender base) { //Disable SSL capabilities
+			if (obj instanceof AbstractHttpSender base) { // Disable SSL capabilities
```

Comment:
Nog een dependency die we in de toekomst moeten zien op te lossen, wanneer we HTTP als module willen loskoppelen.

Reply:
Tja, het is een aanpassing van de defaults, op zich geen probleem aangezien men meestal een request doet naar `http://` endpoints. We hebben al `HasKeystore` en `HasTruststore` interfaces die gebruikt kunnen worden 😄 .

Reply:
Oh ja, dat is zo. Daarmee moet het wel makkelijker worden om deze dependency los te koppelen.